pkg/handler/cli/article_handler.go

```diff
@@ -0,0 +1,207 @@
+package cli
+
+import (
+	"errors"
+	"fmt"
+	"github.com/Masterminds/sprig"
+	"log"
+	"news-aggregator/pkg/model"
+	"news-aggregator/pkg/parser"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"text/template"
+)
+
+// loadData loads articles from the specified files and saves them to the Service.
+func (h *Handler) loadData() error {
+	execDir, err := os.Getwd()
+	if err != nil {
+		log.Fatalf("Error getting current working directory: %v", err)
+	}
+	log.Printf("Current working directory: %s\n", execDir)
+
+	var articles []model.Article
+	files := []string{
+		filepath.Join(execDir, "../../data/abcnews-international-category-19-05-24.xml"),
```

Comment:
can you just read all files in the specified directory instead of hardoding names of each file?